vitrina/datasets/views.py

```diff
@@ -175,3 +178,20 @@ def get_context_data(self, **kwargs):
         }
         context_data.update(extra_context_data)
         return context_data
+
+
+class DatasetMembersView(LoginRequiredMixin, UserPassesTestMixin, ListView):
+    login_url = '/login/'
```

Comment:
```suggestion
```

Do not hardcode urls, and here use `settings.LOGIN_URL` instead.

https://github.com/django/django/blob/stable/3.2.x/django/contrib/auth/mixins.py#L24